guides/common/attributes-base.adoc

```diff
@@ -13,21 +13,49 @@
 :UpgradingDocURL: {BaseURL}Upgrading_and_Updating/{BaseFilenameURL}#
 :ReleaseNotesURL: {BaseURL}Release_notes/{BaseFilenameURL}#
 
+//Added by Marie for GA
 // Repositories and subscriptions (used both in Katello and Satellite guides)
 :SatelliteSub: Red Hat Satellite Infrastructure Subscription
-:RepoRHEL7Server: rhel-7-server-rpms
+:RepoRHEL7Server: rhel-7-server-satellite
+:RepoRHEL8Server: rhel-8-for-x86_64-baseos-rpms 
+// We would probably need to mention also "rhel-8-for-x86_64-appstream-rpms"
 :RepoRHEL7ServerSoftwareCollections: rhel-server-rhscl-7-rpms
 :RepoRHEL7ServerOptional: rhel-7-server-optional-rpms
+// :RepoRHEL7ServerOptional: any equivalent for Sat7 for RHEL 8? - "codeready-builder-for-rhel-8-x86_64-rpms" ?
 :RepoRHEL7ServerAnsible: rhel-7-server-ansible-2.9-rpms
-// For Beta, change to "Beta". For GA releases, change to, for example, "6.8".
-:RepoRHEL7ServerSatelliteServerProductVersion: rhel-7-server-satellite-6.10-rpms
-:RepoRHEL7ServerSatelliteServerProductVersionPrevious: rhel-server-7-satellite-6.9-rpms
-:RepoRHEL7ServerSatelliteCapsuleProductVersion: rhel-7-server-satellite-capsule-6.10-rpms
-:RepoRHEL7ServerSatelliteToolsProductVersion: rhel-7-server-satellite-tools-6.10-rpms
-:RepoRHEL7ServerSatelliteMaintenanceProductVersion: rhel-7-server-satellite-maintenance-6-rpms
+:RepoRHEL8ServerAnsible: ansible-2.9-for-rhel-8-x86_64-rpms 
+:RepoRHEL7ServerSatelliteServerProductVersion: rhel-7-server-satellite-capsule-7.0-rpms
+:RepoRHEL8ServerSatelliteServerProductVersion: satellite-7.0-for-rhel-8-x86_64-rpms
+:RepoRHEL7ServerSatelliteServerProductVersionPrevious: rhel-server-7-satellite-6.10-rpms
+:RepoRHEL7ServerSatelliteCapsuleProductVersion: rhel-7-server-satellite-capsule-7.0-rpms
+:RepoRHEL8ServerSatelliteCapsuleProductVersion: satellite-capsule-7.0-for-rhel-8-x86_64-rpms
+:RepoRHEL7ServerSatelliteMaintenanceProductVersion: rhel-7-server-satellite-maintenance-7.0-rpms
+:RepoRHEL8ServerSatelliteMaintenanceProductVersion: satellite-maintenance-7.0-for-rhel-8-x86_64-rpms
```

Comment:
Once you move them to `attributes-satellite.adoc`, could you use `{ProductVersion}` instead of `7.0`?

Reply:
Yes. I've moved the all these attributes to "attributes-satellite.adoc" and "attributes-katello.adoc". I've tried to apply the {ProductVersion} attribute where possible. Thank you, @ekohl !